include/seqan/stream/formatted_file.h

```diff
@@ -400,6 +400,11 @@ struct FormattedFileContext
     typedef Nothing Type;
 };
 
+template <typename TFormattedFile, typename TStorageSpec>
+struct FormattedFileContext<TFormattedFile const, TStorageSpec> :
```

Comment:
Hm, `TStorageSpec` is something like `Dependent<>` or `Owner<>`. I am not sure about the implications of making tags consts, we used to have problems with const vs. non-const tags. What do you think?